src/app/config/newTeam/newTeamReducer.js

```diff
@@ -0,0 +1,29 @@
+import { GET_USERS_REQUEST_SUCCESS, NEW_TEAM_UNSAVED_CHANGES } from "../constants";
+
+export const initialState = {
+    allPreviewUsers: [],
+    unsavedChanges: false,
+};
+
+const newTeamReducer = (state = initialState, action) => {
```

Comment:
This reducer hold seta of users not teams.the naming is confusing. Im te action you fetched users so it is the same as UsersReducer and should hold state of 
users: 
loadingUsers
etc... 
I am not sure what is the purpose for unsavedChanges ?

Reply:
can you use the current user state ?

Reply:
Although it is storing 'users' the context of these users are users in a basket ready to be added to a new team (that has yet to be created). 

unsavedChanges is used to store whether or not the user has unsaved changes when editing(editing to come) or creating a new team. E.g. if they enter data into the 'team name' field but then click back a popup will appear checking if they want to leave the page or stay on the page as they have unsaved changes. In addition this is used in the bottom bar banner which has the option to save or cancel, if the user has unsaved changes then an exclamation mark in a triangle appears along with the text 'There are unsaved changes' - this is as per UX designs

Reply:
in that situation think `unsavedChanges` should be a local state not global state in store as no other component will have use of this